community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -72,26 +72,24 @@
 
 
 def start_instance_op(inst):
-    return lkp.compute.instances().start(
-        project=lkp.project,
-        zone=lkp.instance(inst).zone,
+    return lookup().compute.instances().start(
+        project=lookup().project,
+        zone=lookup().instance(inst).zone,
         instance=inst,
     )
 
 
 def start_instances(node_list):
     log.info("{} instances to start ({})".format(len(node_list), ",".join(node_list)))
 
-    normal, tpu_nodes = separate(lkp.node_is_tpu, node_list)
-    invalid, valid = separate(lambda inst: bool(lkp.instance), normal)
```

Comment:
why this is removed ?

Reply:
Because it's a nonsense
`bool(lkp.instance)` is attempt to cast function (not result of function call, but function itself to bool), see:

```py
>>> def const_false(): return False
...
>>> bool(const_false)
True
```

As result `invalid` is always empty, all items go into `valid`.
Overall this change does affect behavior, just removes garbage.

Reply:
Looking at code path, only "valid" (i.e. running) instances can be supplied here.